prometheus-hanadb_exporter.spec

```diff
@@ -37,6 +37,7 @@ BuildRequires:  python3-mock
 BuildRequires:  python3-pytest
 %endif
 BuildRequires:  python3-setuptools
+Provides:	hanadb-exporter = 0.6.1
```

Comment:
do you need the version here? also check indentation.

Reply:
in the opensuse example there is the version there

Reply:
could you fix the indentation, though? 😜